### ISSUE TITLE
20-25% cpu usage

### ISSUE THREAD

**Issue by Foul-Tarnished** [picotorrent/picotorrent#1272]
Why is picotorrent using 25% cpu usage ?

i7-8565U 4c/8t

Only one torrent is active ... 
downloading at 0.5-3mb/s, not even seeding.
![image](https://github.com/picotorrent/picotorrent/assets/107711110/b9543fce-088c-4ea2-b603-13ce8767493f)

**Comment by cyberhardt** [comment]
it shouldn't really be using more than a few percents of cpu. could be the antivirus scanning the files as they are being downloaded or something of the sort

**Comment by Foul-Tarnished** [comment]
yeah it's garbage trend micro probably, just strange it's showing the app cpu usage being high, and not the antivirus process
